src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -185,6 +189,30 @@ public function isInitializable(string $class, string $property, array $context
         return false;
     }
 
+    private function extractFromDeclaredType(string $class, string $property)
+    {
+        // to be removed as soon as Symfony bumps the minimum PHP Version to 7.4
```

Comment:
this comment should be removed. Looking for any check on `\PHP_VERSION_ID` is part of our process to bump the min version, so it will be done. The comment would not be checked.